app/src/main/java/io/heckel/ntfy/up/BroadcastReceiver.kt

```diff
@@ -38,6 +38,11 @@ class BroadcastReceiver : android.content.BroadcastReceiver() {
         val repository = app.repository
         val distributor = Distributor(app)
         Log.d(TAG, "REGISTER received for app $appId (connectorToken=$connectorToken)")
+        if (!repository.getEnableUP()) {
+            Log.w(TAG, "Refusing registration because 'EnableUP' is disabled")
+            distributor.sendRegistrationFailed(appId, connectorToken, "EnableUP setting is disabled")
+            return
```

Comment:
IIRC, the reason may be displayed to the user, so let's maybe make this nicer looking? `UnifiedPush is disabled in ntfy` maybe? Could even be a language string.